content/html/concepts/semantic-html/terms/meter/meter.md

```diff
@@ -0,0 +1,38 @@
+---
+Title: '<meter>'
+Description: 'Represents either a fractional value or a scalar value within a known range.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Semantic'
+  - 'Tags'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<meter>` semantic tag is used to represent a fractional or a scalar value within a known range. This form of representation is known as a gauge.
+
+> **Note**: The `<meter>` tag should not be used to indicate progress (as in a progress bar).
+
+## Syntax
+
+The `<meter>` is written as `<meter></meter>` with any contents inserted between the start and end tags:
+
+```html
+<meter>
+    <!-- Meter content goes here. -->
+</meter>
+```
+
+## Example
+
+Below is an example of the `<meter>` tag being used to display disk usage:
+
+```html
+<p>Disk Usage <meter min="100" max="1000" value="80" title="GB"></meter>.</p>
+```
+### Output of The Meter Tag Example
+
+![Output of the meter tag example](https://raw.githubusercontent.com/Codecademy/docs/main/media//meter.png)
```

Comment:
[Suggestion] I highly recommend using a smaller picture so the picture can show up clearly.
```suggestion
![Output of the meter tag example](https://raw.githubusercontent.com/Codecademy/docs/main/media//meter.png)
```

Reply:
Check out this one too @droffilc1

Reply:
How do I do that? @CBID2

Reply:
Consider using [this tool](https://imageresizer.com/) to shrink your image @droffilc1

Reply:
Okay. Thanks!